third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/user-prompt-dialog-main.js

```diff
@@ -109,6 +109,11 @@ function closeWindowClickListener() {
   window.close();
 }
 
+function removeCloseButton() {
+  var closeButton = document.getElementById('close-window');
```

Comment:
nit: "var" is deprecated, "let"/"const" should be used instead. ("const" in this case)